examples/entity-aggregation/README.md

```diff
@@ -115,3 +115,82 @@ Note as well that this example is not intended to be authoritative. Please see
 on writing Spark programs. You may also find
 [this reference on Spark aggregate functions](https://sparkbyexamples.com/spark/spark-sql-aggregate-functions/) helpful.
 
+## Importing customers, rentals and payments from Postgres to MarkLogic
```

Comment:
I think this should not duplicate most of the text above it, but rather explain what it shows in addition to the first example. I'm envisioning something like "This project also contains an example of two levels of joins being aggregated together" (I'm not sure of the correct terminology to use here). "In this example, rentals are aggregated into customers, and payments are also aggregated into rentals, producing the following customer documents (show JSON next). To try this out, run the following Gradle command..."

Basically, you can assume the user has read the stuff above it so we don't need to duplicate so much of the instructions again. 

Alternatively, leave the text as-is but add a new ticket to the 2.2.0 Spark connector release to clean up the README so we don't have so much duplicated text.